src/Proto3/Suite/Class.hs

```diff
@@ -197,6 +197,32 @@ instance HasDefault (Nested a) where
   def = Nested Nothing
   isDefault = isNothing . nested
 
+-- | Used in fields of generated records to represent an unwrapped
+-- 'PackedVec'/'UnpackedVec'
+instance HasDefault (Vector a) where
+  def       = mempty
+  isDefault = null
+
+-- | Used in generated records to represent an unwrapped 'Nested'
+instance HasDefault (Maybe a) where
+  def       = Nothing
+  isDefault = isNothing
+
+-- TODO: Determine if we have a reason for rendering fixed32/sfixed as Fixed
+-- Word32/Int32 in generated datatypes; for other field types, we omit the
+-- newtype wrappers in the type signature but un/wrap them as needed in the
+-- encode/decodeMessage implementations. These Fixed wrappers can probably be
+-- removed and the type interface would be more consistent with other types, but
+-- until that occurs, the following two instances are needed.
+--
+-- Tracked by https://jira.mv.awakenetworks.net/browse/MONAPP-7986
```

Comment:
Perhaps duplicate that ticket to a github issue here as well?

Reply:
Oops, thanks.